addons/payment/models/account_invoice.py

```diff
@@ -56,7 +56,7 @@ def _create_payment_transaction(self, vals):
                 acquirer = payment_token.acquirer_id
 
             if payment_token and payment_token.partner_id != partner:
-                raise ValidationError(_('Invalid token found!'))
+                raise ValidationError(_('The transaction was aborted because you are not the customer of this invoice. Log in as %s to be able to use direct payment methods.') % partner.name)
```

Comment:
Try to stay under the 100 chars limit =]

```suggestion
                raise ValidationError(_(
                    'The transaction was aborted because you are not the customer of this invoice. '
                    'Log in as %s to be able to use direct payment methods.'
                ) % partner.name)
```